format/nexus.py

```diff
@@ -38,6 +38,12 @@
     if "SIT_ROOT" not in os.environ:
         raise
 
+try:
+    import hdf5plugin
+except ImportError:
+    # Import not needed for hdf5-external-filter-plugins
+    pass
```

Comment:
```suggestion
try:
    import hdf5plugin
    assert hdf5plugin
except ModuleNotFoundError:
    # Optional dependency that can also be satisfied by hdf5-external-filter-plugins
    pass
```

* `ImportError` is broader than the more specific `ModuleNotFoundError`
* Resolve the flake8 warning.
* The comment feels slightly misleading